Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+
+COPY <<EOF /etc/yum.repos.d/mongodb-org-6.0.repo
```

Comment:
Source for that is here: 
https://github.com/mongodb/mongodb-atlas-cli/tree/736315ce2a4e0b67fe664709691f95c8baf88716/build/package/docker